lints.toml

```diff
@@ -28,6 +29,18 @@ allow = [
     'clippy::must_use_candidate',
     'clippy::use_self',
     'clippy::wildcard_imports',
+    'clippy::indexing_slicing',
+    'clippy::unreachable',
+    'clippy::missing_inline_in_public_items',
+    'clippy::pattern_type_mismatch',
+    'clippy::exhaustive_enums',
+    'clippy::exhaustive_structs',
+    'clippy::as_conversions',
+    'clippy::shadow_reuse',
+    'clippy::shadow_same',
+    'clippy::integer_arithmetic',
```

Comment:
Can you explain this check please? Does it prohibit use of overflowing operations?

Reply:
Yes